massa-execution-worker/src/speculative_executed_denunciations.rs

```diff
@@ -65,7 +65,11 @@ impl SpeculativeExecutedDenunciations {
         }
```

Comment:
is_de_processed should be renamed is_denunciation_executed

Reply:
Done